### PR TITLE
chore: switch to travis for ci

### PR DESCRIPTION
**Pull request by achingbrain**
See ipfs/aegir#329

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+jobs:
+  include:
+    - stage: check
+      script:
+        - npx aegir commitlint --travis
+        - npx aegir dep-check
+        - npm run lint
+
+     - stage: test
```
Comment (hugomrdias) [inline .travis.yml]:
```suggestion
    - stage: test
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      name: chrome
+      addons:
+        chrome: stable
+      script: npx aegir test -t browser
+
+     - stage: test
```
Comment (hugomrdias) [inline .travis.yml]:
```suggestion
    - stage: test
```

**Comment by hugomrdias** [comment]
can you add this repo to https://github.com/ipfs/aegir/issues/329 please

**Comment by achingbrain** [comment]
Hmm, GitHub's default commit messages for 'suggested edits' don't pass our commit linter.  Great.

**Pull request merged**